### DISCUSSION TITLE
Welcome to RemotePlayWhatever Discussions!

### DISCUSSION THREAD

**Discussion by m4dEngi** [m4dEngi/RemotePlayWhatever#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MisterXar** [comment]
Love the application, have been telling every single person I can.

**Comment by maxkoshe5** [comment]
Thanks for the good stuff m8. It works, but sometimes invite links just won't create.
Played Fallout: New Vegas with a friend, it was fun.

Can it be used for non-steam games? Haven't tried it yet for this purpose.

**Reply by m4dEngi** [reply]
It can be used with non-steam games, but there's still unresolved issue with guest input https://github.com/m4dEngi/RemotePlayWhatever/issues/7
